.gitignore

```diff
@@ -108,6 +108,7 @@ version-info.txt
 
 # Environment variables
 environments/development.env
+environments/dev.env
```

Comment:
Nit picking a bit, but do we need another one here? there is already development.dev

Reply:
The `README` says to create `dev.env` so we should either change the README to be `development.env` or leave this ignore in... Or remove the `development.env` line above? Wdyt?

Reply:
In any case this sort of unrelated change should be in a separate PR, as a courtesy to other developers trying to work out why things changed later on.

Reply:
Good point @cjwatson removed